samples/maps/demo/flowmap-ship-route/demo.js

```diff
@@ -0,0 +1,158 @@
+(async () => {
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Example route of the ship from USA to China'
+        },
+
+        subtitle: {
+            text: `The route through the Suez Canal with a number of travel days
+            between cities`
+        },
+
+        mapNavigation: {
+            enabled: true
+        },
+
+        mapView: {
+            center: [10, 20],
+            zoom: 2.2
+        },
+
+        tooltip: {
+            formatter: function (tooltip) {
+                if (this.series.name === 'Ship Routes') {
+                    return tooltip.defaultFormatter.call(this, tooltip);
+                }
+                return '<b>' + this.point.id + '</b>' + (
+                    this.point.lat ?
+                        '<br>Lat: ' + this.point.lat +
+                        ' Lon: ' + this.point.lon :
+                        ''
+                );
+            }
```

Comment:
Instead of a formatter, use `tooltip.headerFormat` and `tooltip.pointFormat` and set specific `tooltip` options either on `plotOptions` or on the series items.